com.github.sdbg.debug.core/src/com/github/sdbg/debug/core/internal/util/DebuggerUtils.java

```diff
@@ -80,39 +80,7 @@ public static String demangleVmName(String name) {
       return null;
     }
 
-    int atIndex = name.indexOf('@');
```

Comment:
When gwt provides the variable names in source map this should be replaced.  

But for now there's no need to hide it behind a checkbox that is default off.

The code this replaces was modifying any variable with an @ in it so if that didn't cause trouble this won't either.